airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1969,6 +1971,33 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  DpathFlattenFields:
+    title: Dpath Flatten Fields
+    description: A transformation that flatten field values to the to top of the record.
```

Comment:
I _think_ these are the descriptions that show up on the docs site in YAML reference. It could be a good idea to add a larger full example block for how to define this.

Reply:
This is an aspirational and non-blocking comment, do not spend time here yet ;) There are too many components without great docs, no need to polish this one yet